### PR TITLE
Move good wiki pages into the main repo

### PR DESCRIPTION
**Pull request by daniviga**
- absolute links have been fixed
- old and misleading pages deleted

### CODE REVIEW

File: doc/engine/Calculation-Workflow.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Calculation workflows are broken into 6 distinct phases. In order of execution, they are: pre-execute, execute, post-execute, post-process, export, and cleanup. The purpose of each phase is defined below.
+
+### Pre-execute
+
+The pre-execution phase in intended for:
+- parsing and preparing inputs (“unzipping” input XML files into the database, for example)
+- initialising the calculation “work packages” (and saving them to the database)
+- initialising “container” DB records to hold computed results
+
```
Comment (micheles) [inline doc/engine/Calculation-Workflow.md]: Remove every reference to the db here

File: doc/engine/Calculation-Workflow.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+### Execute
+
+The execution phase is where the bulk of the calculation work happens. During this phase, the control node manages task queuing and waits all of the workers to finish the calculation. In this phase the majority of the control node’s time is spent waiting for “completed task” signals from the worker, which indicates that a new task can be enqueued. When there are no more tasks left to enqueue, the control node will simply wait for all of the tasks to finish. (By convention, all calculation results are saved directly to the database by the workers.)
+
```
Comment (micheles) [inline doc/engine/Calculation-Workflow.md]: Remove "By convention, all calculation results are saved directly to the database by the workers."

File: doc/engine/Calculation-Workflow.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- Classical PSHA hazard calculations primarily produce hazard curves. Post processing artefacts for CPSHA include mean hazard curves, quantile hazard curves, hazard maps, and uniform hazard spectra.
+- Event-Based hazard calculations primarily produce stochastic event sets and (optionally) ground motion fields. Post processing artefacts for Event-Based calculations include hazard curves computed from a set of ground motion fields. (This method of producing hazard curves is an alternative to the Classical approach. Given enough ground motion field realisations, in theory the hazard curves produced from GMFs will converge to the curves produced through the Classical approach.)
+
+### Export
+
+If a user specifies the `--exports` option on the command line, all calculation results will automatically be exported from the database to NRML XML.
+
+If the user does not specify this option, it is still possible to export results manually using the `--export-hazard` command line option. (Note: Support will be added in the future for export Risk results as well.)
+
```
Comment (micheles) [inline doc/engine/Calculation-Workflow.md]: Change to
"it is still possible to export results manually using the `--export-output` command line option."

File: doc/engine/Calculation-Workflow.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+### Cleanup
+
+Perform any kind of clean required once all calculations and exports have concluded. Things to clean up might include KVS data, temporary files, or temporary database records.
```
Comment (micheles) [inline doc/engine/Calculation-Workflow.md]: Just write

"Perform any kind of cleanup required once all calculations and exports have concluded. "

**Comment by daniviga** [comment]
fixed comments

**Comment by daniviga** [comment]
more "database" cleanup done

File: doc/engine/Glossary-of-Terms.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  For more information about Celery, see http://celeryproject.org/.
+
+* **worker machine**
+
+  A machine dedicated to run worker processes.
+
+  The concepts of “worker machines” and “worker processes” are often referred to synonymously (as “workers”, for short). Technically, there can be many worker processes running on a single worker machine. The number of worker processes per machine is typically equal to the number of CPU cores available.
+
+* **supervisor**
+
```
Comment (micheles) [inline doc/engine/Glossary-of-Terms.md]: Please remove everything about the supervisor (do a search, there are references even before)

**Comment by micheles** [comment]
LGTM

**Pull request merged**